tests/features/collection/api.feature

```diff
@@ -17,3 +18,31 @@ Feature: Collection API
     Given the following collection:
       | title | EU Interoperability Support Group |
     Then I should have 1 collection
+
+  Scenario: Assign OG ownership during creation of collections through UI
+    Given the following person:
+      | name | Person collection API example |
+    And users:
+      | name                | pass |
```

Comment:
Single user here, and password is not needed, usually we use mail.

> And user
  | name | Daryl Sharman |
  | mail   | daryl.sharman@example.com |
I generated a random name on the internet.